src/main/webapp/i18n/en/quizExercise.json

```
@@ -226,7 +226,17 @@
                 "title": "Short Quiz Title",
                 "questionTitle": "Question Title",
                 "shortQuestionTitle": "Short Question Title"
-            }
+            },
+            "matchLetterCase": {
+                "title": "Match Letter Case",
+                "explanation": "Select if the letter case of the answers provided matters in the evaluation (Example: should 'Answer' and 'answer' be evaluated as the same?)"
+            },
+            "matchPercentage": {
+                "title": "Match Answers Exactly",
+                "explanationTitle": "Similarity %",
```

Remove linebreak in the modal ;)
```suggestion
                "explanationTitle": "Similarity&nbsp;%",
```